modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -290,6 +290,17 @@ private void _validateGetAPIEndpoint(
 					(String)values.get("retrieveType")),
 				APIApplication.Endpoint.RetrieveType.SINGLE_ELEMENT)) {
 
+			if (!Validator.isBlank(pathParameter) &&
```

Comment:
Just a question, is it possible to unset the response schema once the endpoint has been created?

Reply:
Yes, I think so but it's currently unavailable through the UI.